types/interfaces/CreateAssetParams.ts

```diff
@@ -0,0 +1,14 @@
+import { InputSettings } from './InputSettings';
+import { PlaybackIdPolicy } from './PlaybackIdPolicy';
+import { AssetMp4Support } from './AssetMp4Support';
+import { AssetMasterAccess } from './AssetMasterAccess';
+
+export interface CreateAssetParams {
+  input: string | InputSettings[];
+  playback_policy?: PlaybackIdPolicy | (PlaybackIdPolicy)[];
```

Comment:
There is probably a reason for the parenthesis here, right? `PlaybackIdPolicy)[]`?

Reply:
There's no reason, I think this syntactic style I copied over from some generated types based off our Open API spec -- good catch.

Ditto about the conflicting array syntax (`Array<string>` and `string[]`).

I'll make that consistent 👍